docs/en/operations/system-tables/session_log.md

```diff
@@ -0,0 +1,77 @@
+# system.session_log {#system_tables-session_log}
+
+Contains information about all successful and failed login and logout events.
+
+Columns:
+
+-   `type` ([Enum8](../../sql-reference/data-types/enum.md)) — Login/logout result. Possible values:
+    -   `LoginFailure` — Login error.
+    -   `LoginSuccess` — Successful login.
+    -   `Logout` — Logout from the system.
+-   `session_id` ([UUID](../../sql-reference/data-types/uuid.md)) — Session id.
+-   `session_name` ([String](../../sql-reference/data-types/string.md)) — Session name.
```

Comment:
The columns `session_id`, `session_name`, and `changed_settings` have changed, see https://github.com/ClickHouse/ClickHouse/pull/30882

Reply:
@vitlibar Ok. I renamed the columns.